### ISSUE TITLE
V6 Fix Error c1-2858-3

### ISSUE THREAD

**Issue by PedroMendesAraujo** [TheOfficialFloW/Adrenaline#187]
I use a Fat PS Vita with HenKAKU Enso. I first installed V6 and it crashed when trying to open the Memory Stick directory because it contained .cso files. I uninstalled it on the bubble, and installed v6 fix using VitaShell. 
The only problem is that Adrenaline won't start... it shows the Adrenaline logo and then fades to a black screen for about a minute, and then displays Error c1-2858-3 and returns to main screen.
I have the PSPEMUCFW kernel under the config file.

I tried uninstalling v6 fix and installing it again, and the problem persists. It also persists if I try to install v6 again. It won't even give me the option to install the firmware, only a black screen.

Is there any residual file I need to uninstall? I am very new to PS Vita homebrew, so it might be something very simple to fix. I am using an original PS Vita memory card.

Thanks in advance, and congratulations for your work.

**Comment by PedroMendesAraujo** [comment]
Nevermind, I deleted everything in the pspemu folder, reinstalled the v6 fix version and restarted my Vita, and now it works, and I've even tested .cso files and they work as well!

Thanks for your work.

**Comment by rsn8887** [comment]
Remember you can close the issue yourself if you opened it.

**Comment by eivanhoe** [comment]
Did you manage to run large ISO games?

**Comment by PedroMendesAraujo** [comment]
Yes, I've opened both PSX Games and PSP Games larger than 1.3GB (cso)
